src/constants/button.js

```diff
@@ -1,12 +1,16 @@
 /* @flow */
 export const BUTTON_LABEL = {
-    CONNECT: ('connect' : 'connect')
+    CONNECT: ('connect' : 'connect'),
```

Comment:
we can remove the existing one if it is not needed.

Reply:
done